scapy/layers/ntp.py

```diff
@@ -1,80 +1,1893 @@
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                          Reference ID                         |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  +                     Reference Timestamp (64)                  +
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  +                      Origin Timestamp (64)                    +
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  +                      Receive Timestamp (64)                   +
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  +                      Transmit Timestamp (64)                  +
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  .                                                               .
+    #  .                    Extension Field 1 (variable)               .
+    #  .                                                               .
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  .                                                               .
+    #  .                    Extension Field 2 (variable)               .
+    #  .                                                               .
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                          Key Identifier                       |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #  |                                                               |
+    #  |                            dgst (128)                         |
+    #  |                                                               |
+    #  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    #
+    #                  Figure 8: Packet Header Format
+    #________________________________________________________________________
+    #
+
+    name = "NTPHeader"
+    fields_desc = [
```

Comment:
Could you use " instead of ' for string ? It will consistent with the rest of your patch.